airflow/providers/microsoft/azure/operators/container_instances.py

```diff
@@ -126,11 +126,11 @@ def __init__(
         self,
         *,
         ci_conn_id: str,
-        registry_conn_id: str | None,
         resource_group: str,
         name: str,
         image: str,
         region: str,
+        registry_conn_id: str | None = None,
```

Comment:
mmm we are to considered this as breaking change since this changes the order of parameters and will break code of users that passes positional arguments

Reply:
It's keyword only - see '*'

Reply:
ahh :)